cfg/cs2fixes/cs2fixes.cfg

```diff
@@ -46,6 +46,10 @@ cs2f_vote_max_nominations		10		// Number of nominations to include per vote, out
 // User preferences settings
 cs2f_user_prefs_api				""		// User Preferences REST API endpoint
 
+// Extra addon settings
+cs2f_extra_addon 				""		// The workshop ID of an extra addon to mount and send to clients
+cs2f_extra_addon_timeout		10		// How long in seconds until clients are timed out in between connects for the extra addon, requires cs2f_extra_addon to be used
```

Comment:
10 seconds feels short… Is there any impact to making this longer? (Say, a minute.)

It just feels a little clunky to reject players for being away for 10 seconds, especially when it’s (theoretically) caused by:
- Slow computers
- First-timers clicking “accept download” too slow
- Updated addons on slow connections
- Acts of god

Reply:
If the player already has the addon, the reconnect is nearly instantaneous so it isn't a big issue. One of the reasons for the timeout is that we've had players who would deny the download prompts then proceed to complain about error models and/or crash.